vault/resource_identity_entity_alias.go

```diff
@@ -108,6 +119,11 @@ func identityEntityAliasUpdate(d *schema.ResourceData, meta interface{}) error {
 	if canonicalID, ok := d.GetOk("canonical_id"); ok {
 		data["canonical_id"] = canonicalID
 	}
+	if customMetadata, ok := d.GetOk("custom_metadata"); ok {
+		data["custom_metadata"] = customMetadata
+	} else {
+		data["custom_metadata"] = make(map[string]interface{})
+	}
```

Comment:
This feels like it could just be:

```suggestion
	data["custom_metadata"] = d.Get("custom_metadata").(map[string]interface{})
```